Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -25,19 +28,25 @@ def __init__(self, comport):
         # initialize software components
         self.reward_manager = RewardManager(comport)
         self.videoAnalyser = Video_Analyzer()
+        self.mouse_id="1780"
+        self.videoAnalyser = Video_Analyzer(self.mouse_id)
```

Comment:
You are instantiating the video analyzer twice. see two lines above. I think you want only one of them, no?

Reply:
resolved

Reply:
OK.